packages/integrations/components/pico/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+
+const Pico = (props) => {
+  const {
+    pageInfo,
+    publisherId,
+  } = props;
+
+  // Ensure page info has the right format.
+  const picoPageInfo = {
+    article: pageInfo.article,
+    post_id: pageInfo.postId,
+    post_type: pageInfo.postType,
+    resource_ref: pageInfo.resourceRef,
+    taxonomies: {},
+    url: window.location.href,
+  };
+
+  const [picoLoaded, setPicoLoaded] = useState(false);
+
+  // On component hydration, add an event listener to watch for the script's init event.
+  useEffect(() => {
+    window.document.addEventListener(
+      'pico-init',
+      () => {
+        setPicoLoaded(true);
+      }
+    );
+  });
```

Comment:
Provide a `[]` as the second parameter here to avoid unnecessarily calling this hook again.